### ISSUE TITLE
We need to consider equal-preference voting

### ISSUE THREAD

**Issue by dwsinger** [w3c/w3process#115]
We seem to have a strong desire to have it, but the open-source software doesn't support it. If we can't have it, do we have the right voting system?

**Comment by michaelchampion** [comment]
>  If we can't have it, do we have the right voting system?

No.  I will be opposing removing the “one vote per available seat” language until this is resolved. I supported STV in Process 2017 under the naive assumption that the team implemented the process as written with the one vote per available seat constraint. During the years of discussion, I can’t recall or find anyone mentioning that the STV software being used was inconsistent with the widely used practice of voting for slates of candidates rather than forcing an arbitrary ranking.  If the process as written is inconsistent with STV theory or software implementation, I’d prefer the old system.

**Comment by chaals** [comment]
If we don't have it we have a suboptimal version of the right voting system, but we could improve it significantly by implementing equal ranking.

**Comment by Ad-Huikeshoven** [comment]
Hi. You are looking for open source STV software that allows equal rankings. You can consider https://github.com/nealmcb/flexstv. This software has been used several times in elections for Affilisate Selected Board Seats of the Wikimedia Foundation. Have a nice day.
